content/en/guide/v10/hooks.md

```diff
@@ -514,3 +535,98 @@ const App = props => {
 ```
 
 > This hook was introduced with Preact 10.11.0 and needs preact-render-to-string 5.2.4.
+
+### useDebugValue
+
+Displays a custom label for use in the Preact DevTools browser extension. Useful for custom hooks to provide additional context about the state or value they represent.
+
+```jsx
+import { useDebugValue, useState } from 'preact/hooks';
+
+function useCount() {
+	const [count, setCount] = useState(0);
+	useDebugValue(count > 0 ? 'Positive' : 'Negative');
+	return [count, setCount];
+}
+```
+
+In your devtools, this will display as `useCount: "Positive"` or `useCount: "Negative"`, whereas previously it would've been just `useCount`.
+
+Optionally, you can also pass a function as the second argument to `useDebugValue` for use as the "formatter".
+
+```jsx
+import { useDebugValue, useState } from 'preact/hooks';
+
+function useCount() {
+	const [count, setCount] = useState(0);
+	useDebugValue(count, c => `Count: ${c}`);
+	return [count, setCount];
+}
+```
```

Comment:
Side note: the formatter arg is lazy in React, hence why it exists, but our impl is very much not. It immediately calls the formatter upon ever render of the component, regardless of whether devtools are inspecting the element.

Not sure if that was intentional or not.